crates/oxc_transformer/src/react_jsx/options.rs

```diff
@@ -28,3 +30,30 @@ impl ReactJsxRuntime {
         matches!(self, Self::Automatic)
     }
 }
+
+impl ReactJsxOptions {
+    /// Scan through all comments and find the following pragmas
+    ///
+    /// * @jsxRuntime classic / automatic
+    ///
+    /// The comment does not need to be a jsdoc,
+    /// otherwise `JSDoc` could be used instead.
+    ///
+    /// This behavior is aligned with babel.
+    pub(crate) fn with_comments(mut self, semantic: &Semantic) -> Self {
+        for (_, span) in semantic.trivias().comments_spans() {
+            let comment = span.source_text(semantic.source_text());
+            // strip leading jsdoc comment `*` and then whitespaces
+            let comment = comment.strip_prefix('*').unwrap_or(comment).trim_start();
+            // strip leading `@`
+            let Some(comment) = comment.strip_prefix('@') else { continue };
+            // read jsxRuntime
+            match comment.strip_prefix("jsxRuntime").map(str::trim) {
+                Some("classic") => self.runtime = ReactJsxRuntime::Classic,
```

Comment:
Do we need to store every option in a struct? Can it be stored in self.options.xxx? We still have a lot of options that we don't support yet, and I'm kind of curious about that

Reply:
I don't get your question, this is saving the value into `ReactJsxOptions`.

Reply:
Ah. I see, Thank you!